R/summarize_copilot.R

```diff
@@ -98,6 +99,12 @@ team_cycle_class_participation <- function(tcc,
                                            triton.classroom,
                                            neptune.participant,
                                            neptune.participant_data) {
+  # Args:
+  #   tcc - see team_cycle_class()
+  #   triton.cycle - from db, include all cycles from tcc
+  #   triton.classroom - from db, include all classes from tcc
+  #   neptune.participant - from db, include all ppts from...
+  #   neptune.participant - from db, include all relevant survey responses
```

Comment:
Nit: I almost always use saturn data instead of neptune participant data. Would be nice for the argument name to reflect this possibility; e.g., call it `survey_responses` and include a note that `neptune.participant_data` can also be used in a pinch, rather than the reverse